.github/workflows/ci.yml

```diff
@@ -153,10 +152,13 @@ jobs:
 
   deploy:
     name: Deploy
-    needs: [build, storybook]
+    # needs: [build, storybook]
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
```

Comment:
Don't care about the storybook, but no longer depends on `build`?

I see you added the code checkout step, but why? Need to have the source in order to push stuff?

Reply:
yes, that was a test!
forgot to uncomment